Thales/thales_safenet/ingest/parser.yml

```diff
@@ -0,0 +1,35 @@
+name: thales_safenet
+pipeline:
+  - name: deserialize
+    description: ''
+    filter: ''
+    external:
+      name: json.parse-json
+      properties:
+        input_field: '{{original.message}}'
+        output_field: message
+  - name: general_value_setter
+    description: '' 
+    filter: ''
+    external: null
+stages:
+  general_value_setter:
+    actions:
+      - set:
+          event.module: 'safenet'
+          event.category: 'authentification'
+          event.code: '{{deserialize.message.details.actionText}}'
+          event.type: '{{deserialize.message.details.type}}'
+          event.action: '{{deserialize.message.details.action}}'
+          event.reason: '{{deserialize.message.details.message}}'
+          user.name: '{{deserialize.message.context.principalId}}'
+          source.ip: "{{deserialize.message.context.originatingAddress.replace(\" \", \"\")}}"
```

Comment:
Can only contain one IP 

You will have to use something like https://www.elastic.co/guide/en/ecs/current/ecs-host.html#field-host-ip which is an array